src/app/news/release-page/release-organizer.ts

```diff
@@ -5,7 +5,7 @@ export const MINORS: { [major: string]: string[] } = {};
 export const PATCHES: { [minor: string]: string[] } = {};
 
 organize();
-sortAll();
+// sortAll();
```

Comment:
Can we remove instead of commenting out?

Reply:
Yep, I just pushed an update. I wanted to get buy in from the team first, then remove and rebase.